astroplan/tests/test_core.py

```diff
@@ -155,3 +152,591 @@ def test_FixedTarget_ra_dec():
                                                         'SkyCoord')
     assert vega.coord.dec == vega_coords.dec == vega.dec, ('Retrieve Dec from '
                                                            'SkyCoord')
+
+def test_sunrise_sunset_equator():
+    '''
+    Check that time of sunrise/set for an observer on the equator is
+    consistent with PyEphem results (for no atmosphere/pressure=0)
+    '''
+    lat = '00:00:00'
+    lon = '00:00:00'
+    elevation = 0.0 * u.m
+    pressure = 0 * u.bar
+    location = EarthLocation.from_geodetic(lon, lat, elevation)
+    time = Time('2000-01-01 12:00:00')
+    obs = Observer(location=location, pressure=pressure)
+    astroplan_next_sunrise = obs.sunrise(time, which='next').datetime
+    astroplan_next_sunset = obs.sunset(time, which='next').datetime
+
+    astroplan_prev_sunrise = obs.sunrise(time, which='previous').datetime
+    astroplan_prev_sunset = obs.sunset(time, which='previous').datetime
+
+    # Run print_pyephem_sunrise_sunset() to compute analogous
+    # result from PyEphem:
+    pyephem_next_sunrise = datetime.datetime(2000, 1, 2, 6, 3, 39, 150790)
+    pyephem_next_sunset = datetime.datetime(2000, 1, 1, 18, 3, 23, 676686)
+    pyephem_prev_sunrise = datetime.datetime(2000, 1, 1, 6, 3, 10, 720052)
+    pyephem_prev_sunset = datetime.datetime(1999, 12, 31, 18, 2, 55, 100786)
+
+    # Typical difference in this example between PyEphem and astroplan
+    # with an atmosphere is <2 min
+    threshold_minutes = 8
+    assert (abs(pyephem_next_sunrise - astroplan_next_sunrise) <
+            datetime.timedelta(minutes=threshold_minutes))
+    assert (abs(pyephem_next_sunset - astroplan_next_sunset) <
+            datetime.timedelta(minutes=threshold_minutes))
+    assert (abs(pyephem_prev_sunrise - astroplan_prev_sunrise) <
+            datetime.timedelta(minutes=threshold_minutes))
+    assert (abs(pyephem_prev_sunset - astroplan_prev_sunset) <
+            datetime.timedelta(minutes=threshold_minutes))
+
+def test_sunrise_sunset_equator_fast_and_slow():
+    '''
+    Check that time of sunrise/set for an observer on the equator is
+    consistent with PyEphem results (for no atmosphere)
+    for the fast method assuming pressure=0, versus with less fast
+    method where pressure=None.
+    '''
+    lat = '00:00:00'
+    lon = '00:00:00'
+    elevation = 0.0 * u.m
+    pressure = 0 * u.bar
+    location = EarthLocation.from_geodetic(lon, lat, elevation)
+    time = Time('2000-01-01 12:00:00')
```

Comment:
For this test I think it's insufficient to only do lat/lon=0,0 and 2000-1-1.  Those are exactly where you expect it to be simplest/most accurate, and we want to test where it's most likely to go *wrong*.  (A bit more about this in my PR comments)